x-pack/plugins/security_solution/public/common/components/alert_count_by_status/alert_count_by_rule_by_status.tsx

```diff
@@ -200,7 +200,9 @@ export const getTableColumns: GetTableColumns = (openRuleInTimelineWithAdditiona
     render: (count: number, { ruleName }) => (
       <EuiLink
         disabled={count === 0}
-        onClick={() => openRuleInTimelineWithAdditionalFields(ruleName)}
+        onClick={async () => {
+          await openRuleInTimelineWithAdditionalFields(ruleName);
+        }}
```

Comment:
Should the `GetTableColumns` type up on line `50` be updated as well (along with marking the arg passed to `getTableColumns` on `84` as `async`)?

```
type GetTableColumns = (
  openRuleInTimelineWithAdditionalFields: (ruleName: string) => Promise<void>
) => Array<EuiBasicTableColumn<AlertCountByRuleByStatusItem>>;
```

```
return getTableColumns(async (ruleName: string) => {
```

Reply:
nope, because async is only the callback we pass to the `onClick`, while `GetTableColumns` returns an array of columns, so it's synchronous action